controllers/admin/AdminImportController.php

```diff
@@ -2992,10 +3002,14 @@ protected function customerImportOne($info, $default_language_id, $id_lang, $sho
         }
 
         if (!$res) {
-            $this->errors[] = sprintf(
-                Tools::displayError('%1$s (ID: %2$s) cannot be '.($validateOnly?'validated':'saved')),
-                $info['email'],
-                (isset($info['id']) && !empty($info['id']))? $info['id'] : 'null'
+            $this->errors[] = $this->trans(
+                '%1$s (ID: %2$s) cannot be %3$s',
```

Comment:
Easier to translate: `Email address %1$s (ID: %2$s) cannot be %3$s`.

In fact, this should split into two strings:
* `Email address %1$s (ID: %2$s) cannot be validated`.
* `Email address %1$s (ID: %2$s) cannot be saved`.

`($validateOnly?'validated':'saved')` was _really_ a bad idea :)